action.yml

```diff
@@ -83,6 +83,10 @@ inputs:
     description: 'Action is waiting for testing farm to finish or until timeout is reached. Value is in minutes.'
     required: false
     default: '480'
+  tmt_hardware:
+    description: 'Define hardware requirements for the testing environment as a JSON string. See https://tmt.readthedocs.io/en/latest/spec/hardware.html'
+    required: false
+    default: '{}'
```

Comment:
You probably don't want to set a default value. That would mean that every time unsafeNewRequest is used.

```suggestion
```